src/amo/actions/reviews.js

```diff
@@ -3,12 +3,14 @@ import { SET_REVIEW } from 'amo/constants';
 export const setReview = (review, reviewOverrides = {}) => ({
```

Comment:
Is `reviewOverrides` needed? Couldn't the caller do this just as easily? Looks like it's only for tests.

```jsx
const action = setReview({ ...fakeReview, isLatest: true });
// rather than
const action = setReview(fakeReview, { isLatest: true });